File: C_Program/while1.c
//wap to accept your name and print your name 5 times.

#include<stdio.h>
int main()
{
	int i=1;
	char name[20];
	printf("\n enter your name: ");
	scanf("%s",&name);
	while(i<=5){
		printf("\n name = %s",name);
		i=i+1;
	}
}

File: Java_Programs/While1.java
//wap to accept your name and print your name 5 times.
import java.util.*;
class While1
{
	public static void main(String []args)
	{
		int i=1;
		String name;

		Scanner s=new Scanner(System.in);

		System.out.println("Enter your name: ");
		name = s.next();

		while(i<=5)
		{
			System.out.println("Name: " + name);
			i++;
		}

	}
}